packages/libticalcs2.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Libticalcs2 < Package
+  description 'Libticalcs2 offers the library used to properly manipulate files on TI calculators.'
```

Comment:
Isn't this the same description of libtifiles2?

Reply:
Yeah. Let me change that.